resources/models/utils.py

```diff
@@ -108,10 +110,12 @@ def send_respa_mail(email_address, subject, body, html_body=None):
 
     notification_logger.info('Sending notification email to %s: "%s"' % (email_address, subject))
 
+    subject, from_email, to = subject, from_address, email_address
+    text_content = body
+    msg = EmailMultiAlternatives(subject, text_content, from_email, [to], attachments=attachments)
```

Comment:
Why not just use the original variables as parameters to EmailMultiAlternatives instead of mapping them to new names?

Reply:
I assume this was a remain from refactoring the `send_respa_mail()` method. Changed.